pophory-iOS/Screen/Auth/Views/PickAlbumCoverView.swift

```diff
@@ -72,24 +90,78 @@ extension PickAlbumCoverView {
             $0.width.equalTo(convertByWidthRatio(254))
             $0.height.equalTo(convertByHeightRatio(50))
         }
+        
+        initialAlbumSelection()
+    }
+    
+    // MARK: - @objc
+    
+    @objc private func swipe(_ gesture: UISwipeGestureRecognizer) {
+        switch gesture.direction {
+        case .left:
+            scrollAlbumCover(next: true)
+        case .right:
+            scrollAlbumCover(next: false)
+        default:
+            break
+        }
+    }
+    
+    // MARK: - Private Methods
+    
+    private func scrollAlbumCover(next: Bool) {
+        guard let indexPath = lastSelectedItemIndex else { return }
+        var item = indexPath.item
+        
+        if next && item + 1 < selectButtonCollectionView.numberOfItems(inSection: 0) {
+            item += 1
+        } else if !next && item - 1 >= 0 {
+            item -= 1
+        } else {
+            return
+        }
```

Comment:
if문의 반복을 이렇게 줄여보는 방법도 있어

```suggestion
        var item = indexPath.item
        var totalCount = selectButtonCollectionView.numberOfItems(inSection: 0)
        
        if next ? (item + 1 < totalCount) : (item - 1 >= 0) {
            item = next ? (item + 1) : (item - 1)
        } else {
            return
        }
```